install/kubernetes/helm/istio/charts/grafana/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.security.enabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: grafana
+  labels:
+    app: grafana
+type: Opaque
+data:
+  username: {{ .Values.security.adminUser | b64enc | quote }}
+  password: {{ .Values.security.adminPassword | b64enc | quote }}
+{{- end -}}
```

Comment:
![image](https://user-images.githubusercontent.com/4461983/42917471-380cbf68-8b3c-11e8-9831-352454e46a82.png)

Kill the red enter

Reply:
done